Productsの実装

yamachu opened the pull request:
close: https://github.com/oystersjp/oystersjp.github.io/issues/98

yamachu committed:
feat: Productの型を定義して旧ページの要素を定義する

yamachu committed:
misc: productsのimageを追加

yamachu committed:
feat: Productsページの実装

yamachu committed:
fix: tagの間隔が狭まっているのでCSSあてる

YuMuuu commented:
`RangeError: WebAssembly.Memory(): could not allocate memory`
CIガチャっぽい

https://github.com/oystersjp/oystersjp.github.io/pull/126/checks?check_run_id=2319747585#step:6:19

↑違った

YuMuuu commented:
rerunしておきますね

YuMuuu commented:
むむむ

YuMuuu commented:
https://github.com/cyrilwanner/next-optimized-images/issues/196#issuecomment-691661664

https://github.com/cyrilwanner/next-optimized-images/issues/186#issuecomment-753632391

この変が原因かもしれないです

mii288 committed:
perf: resize images

mii288 committed:
fix: スペースの追加

mii288 commented:
https://github.com/oystersjp/oystersjp.github.io/pull/128 直してみました
画像リサイズしたらエラーでなくなりました 🤔

mii288 commented on styles/custom.css:
```diff
@@ -49,6 +49,14 @@
+  Products section
+==================================================================*/
+
+.products-tags:not(:last-child) {
```
poem: 書き手の好みですが、 :not 使うと詳細度高くなるので、隣接セレクタでかいても良いかもしれないですね。
（このデザインいつまで使うのかという問題もありますがｗ）

```css
.products-tags + .products-tags {
  margin-left: 4px;
}
```

yamachu replied:
未だに詳細度の優先度わかってないからなるほどなとなった
高すぎると上書きづらいし良さそう

yamachu committed:
Merge pull request #128 from oystersjp/feat/products-ci

Productsの画像のリサイズ（CIのエラー修正）

yamachu committed:
refactor: CSSの詳細度を落として同じデザインになるようにする

https://github.com/oystersjp/oystersjp.github.io/pull/126#discussion_r613591001
詳細度高いと上書きづらいし良さそうに思える
